### ISSUE TITLE
Get some acceptance tests set up

### ISSUE THREAD

**Issue by courajs** [hoodiehq/hoodie-admin#9]
One challenge is setting up the server to run during tests. In Ember this is often done by mocking out your API. For this we may just want to run an actual hoodie server.
Then we should have tests walking through each important user flow. We probably need to create a good list of user flows :)

**Comment by gr2m** [comment]
Good call. Agree with simply starting a hoodie server for acceptance tests. We can start it in memory and seed some data as we need it
